### PR TITLE
Blood Vasculature Organ Crosswalk updated to v1.3

### PR DESCRIPTION
**Pull request by emquardokus**
update Blood Vasculature Organ Crosswalk updated to v1.3

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -17,5 +17,8 @@
     "node-dist-vis",
     "lib:common",
     "ccf-rui"
+  ],
```
Comment (bherr2) [inline .vscode/settings.json:20]: @emquardokus please revert this change.

**Review by emquardokus — COMMENTED** [review]
I could not see a change, but hit revert button at line 20
line 84 is the change I made from v1.2 to v1.3

**Review by emquardokus — COMMENTED** [review]
sorry I was in the vccf.yaml, not settings.json

File: .vscode/settings.json (hunk 1 above)
Comment (emquardokus) [reply .vscode/settings.json:20]: reverted settings.json line 20

**Review by emquardokus — COMMENTED** [review]
OK on line 20 settings.json
